src/BusinessTime.ts

```diff
@@ -48,6 +51,20 @@ export class BusinessTime {
         return true
     }
 
+    isBusinessDay(): boolean {
+        for (const constraint of this.constraints) {
```

Comment:
Minor, but may be easier to read on one line, though that's maybe subjective 😄  `this.constraints.every(constraint => isBusinessDay(constraint) && constraint.isBusinessTime(this.getMoment))`